iml-wasm-components/iml-action-dropdown/src/api_transforms.rs

```diff
@@ -42,10 +43,34 @@ pub fn group_actions_by_label(objects: Vec<AvailableAction>, records: RecordMap)
         })
 }
 
+/// Sort items by display_group, then by display_order. Mark the last item in each group
+pub fn sort_actions(mut actions: Vec<AvailableAction>) -> Vec<AvailableAction> {
```

Comment:
Why do we need to know if something is last?

Reply:
It's used to create a divider in the dropdown.

Reply:
Example? This seems unnecessary. I.E. We should be able to partition actions into resource groups, and we already do with ActionMap.

Reply:
Ok, i'm building out the elements now so I'll do it without that property and remove it.

Reply:
Is this being called over keys on the ActionMap? If so, you should be able to iterate the HashMap which will tell you where each divider goes.

Reply:
Right, yeah that makes sense so i'll remove the `last` property.